rfc8446.xml

```diff
@@ -2817,7 +2820,7 @@ handshake with a "bad_certificate" alert.  SHA-1 is deprecated, and it
 is RECOMMENDED that any endpoint receiving any certificate which it
 would need to validate using any signature algorithm using a SHA-1
 hash abort the handshake with a "bad_certificate" alert. For clarity,
-this means that endpoints MAY accept these algorithms for
+this means that endpoints can accept these algorithms for
```

Comment:
These three are potentially controversial, since we spent a lot of time in the WG on these topics.
That said, it's not clear there is actually any normative requirement here; we're just describing the only two choices (for the first two), or clarifying the previous normative language.

Reply:
So I think that you’re right that this is a statement of the two possible things this could do.
I mean, we could say you MUST do X.

Reply:
Fair enough; we should not make the first two changes.  The last one is a slightly different scenario, and I will think about it some more.